README.md

```diff
@@ -141,6 +141,33 @@ _(e.g. Maven artifacts)_
 
 Third party (external) artifacts can be brought in with systems such as [`rules_jvm_external`](https://github.com/bazelbuild/rules_jvm_external) or [`bazel_maven_repository`](https://github.com/square/bazel_maven_repository) or [`bazel-deps`](https://github.com/johnynek/bazel-deps), but make sure the version you use doesn't naively use `java_import`, as this will cause bazel to make an interface-only (`ijar`), or ABI jar, and the native `ijar` tool does not know about kotlin metadata with respect to inlined functions, and will remove method bodies inappropriately.  Recent versions of `rules_jvm_external` and `bazel_maven_repository` are known to work with Kotlin.
 
+
+# Development Setup Guide
+To use the rules directly from git, an additional dependency downloads are required. 
```

Comment:
s/git/the rules_kotlin workspace (i.e. not the release artifact)/

"an additional...downloads" - plurals should agree.

Reply:
done